src/redux/reducers/employer.ts

```diff
@@ -1,27 +1,53 @@
+import { employerInfoFromStorage } from '../saga/employer'
 import {
-  employerActions,
+  EmployerActions,
   REGISTER_EMPLOYER_REQUEST,
   REGISTER_EMPLOYER_SUCCESS,
   REGISTER_EMPLOYER_FAIL,
+  LOGIN_EMPLOYER_REQUEST,
+  LOGIN_EMPLOYER_SUCCESS,
+  LOGIN_EMPLOYER_FAIL,
+  LOGOUT_EMPLOYER,
 } from '../types'
 
 const initialState = {
-  credentials: '',
+  info: {},
+  credential: { email: '', password: '' },
+  employerInfo: employerInfoFromStorage,
   loading: false,
   error: null,
 }
 
-const test = (state = initialState, action: employerActions) => {
+const employer = (state = initialState, action: EmployerActions) => {
   switch (action.type) {
     case REGISTER_EMPLOYER_REQUEST:
-      return { ...state, loading: true, credentials: '' }
+      // if ('email' && 'password' in action.payload) {
+      return {
+        ...state,
+        loading: true,
+        // info: {},
+        //@ts-ignore
+        email: action.payload.email,
+        //@ts-ignore
+        password: action.payload.password,
```

Comment:
I see you've changed the `initialState` to use `credential`. Then you have to change the way of updating state also. In this case:
`return { ...state, loading: true, credential: { email: action.payload.email, password: action.payload.password } }`

because you action returns an action object: `{ type: REGISTER_EMPLOYER_REQUEST, payload: { info, email, password }` }`

Reply:
It will change the request body to be something that it shouldn't be. I use it in the login request but I'm going to remove it...